README.md

```diff
@@ -1,5 +1,11 @@
 # Azure Pipelines Agent
 
+## Announcement -  `AZP_AGENT_USE_LEGACY_HTTP` agent knob future deprecation
+
+We are working on pipeline agent migration to .NET 6. One of the side effect of this migration is that the legacy HTTP handler will be no longer available for use due to changes in the .NET framework itself.
```

Comment:
'.NET Framework' refers to the Windows bundled (and Windows-only) version of .NET. You could say '.NET runtime' instead of '.NET framework'